content/learn/beginner/datatypes-1.md

```diff
@@ -0,0 +1,131 @@
+Please note that not all escape sequences are supported by Discord.
+
+### Raw String Literals
+
+It should become relatively clear that a lot of new lines and other special characters can make a quoted string quite
+hard to read. To make this easier, you can use backticks (`` ` ``) to create a *raw string literal*. This enables you to
+*literally* press enter to insert a new line in your string, without having to use the `\n` escape sequence, like so:
+
+```txt
+`This is my
+cool multiline string!
+
+Look at all the new lines!`
+```
+
+To insert a backtick in a raw string literal, however, you must use its ASCII code, which is `\x60`. Escape sequences as
+in quoted strings are not supported.
+
+## Numbers
+
+Numeric values can be represented in two ways, using integers (whole numbers) and floating-point numbers (numbers with a
+decimal point). In the bot's template actions, you can create an integer by simply writing a number, such as `5`. For
+floating-point numbers, you can add a decimal point, like so: `5.0`.
+
+### Integers
+
+In the bot's template actions, integers are represented as 64-bit signed integers. This means that you can store numbers
+from `-9223372036854775808` to `9223372036854775807`. If you try to store a number outside this range, the bot will
+return an error.
+
+There are several ways to define an integer, specifically:
+
+1. As a base-10 number, such as `42`. This will mean what you think, the number forty-two.
+2. As a base-16 number, such as `0x2A`. This is the [hexadecimal representation][hex] of the number forty-two.
+3. As a base-8 number, such as `052`. This is the [octal representation][oct] of the number forty-two.
+4. As a base-2 number, such as `0b101010`. This is the [binary representation][bin] of the number forty-two.
+
+[hex]: https://en.wikipedia.org/wiki/Hexadecimal
+[oct]: https://en.wikipedia.org/wiki/Octal
+[bin]: https://en.wikipedia.org/wiki/Binary_number
+
+### Floating-Point Numbers
+
+We represent floating-point numbers as 64-bit IEEE-754 floating-point numbers. This means that you can store numbers
+with a precision of about 15 decimal places. If you try to store a number with more precision, the bot will round it to
+the nearest representable number.
+
+There are a lot of ways to define a floating-point number, but the most common way is to use the decimal point, such as
+`3.14`. For a full list of ways to define a floating-point number, you can refer to the
+[Go documentation](https://golang.org/ref/spec#Floating-point_literals).
+
+### Imaginary Numbers
+
+Go (the language the bot is written in) also supports imaginary numbers (and by extension complex numbers).
+
+Defining them is quite similar to the formats discussed above, but with the addition of the imaginary unit `i`.
+Please refer to the [Go documentation](https://golang.org/ref/spec#Imaginary_literals) for more information.
```

Comment:
I would prefer to remove this section entirely. It is wholly out of place in an introductory document and better relegated to some deep, dusty reference section.